dist/google-music.js

```diff
@@ -338,9 +352,9 @@ proto.hooks = {
             var durationStr = that.doc.getElementById(SELECTORS.playback.sliderId).getAttribute('aria-valuemax');
             var duration = parseInt(durationStr, 10);
 
-            title = (title) ? title.innerText : 'Unknown';
-            artist = (artist) ? artist.innerText : 'Unknown';
-            album = (album) ? album.innerText : 'Unknown';
+            title = (title) ? (title.innerText || title.textContent) : 'Unknown';
```

Comment:
Darn it, it is updated in `main.js` but I didn't push the dist for some reason......